### PR TITLE
extend proptests for sorting and merge

### PR DESCRIPTION
**Pull request by PSeitz**
extend proptests for sorting and merge
test multivalue fastfields
#1100

### CODE REVIEW

File: src/indexer/index_writer.rs [hunk 1]
```diff
@@ -1401,15 +1434,41 @@ mod tests {
                     .map(move |doc| ff_reader.get(doc))
             })
             .collect();
+        ids.sort();
```
Comment (fulmicoton) [inline src/indexer/index_writer.rs]: we should probably disable the sorted is not enabled? (that requires disabling it in the expected_ids function too)

Comment (PSeitz) [reply src/indexer/index_writer.rs]: the sort is required because the order is not kept with the merges. I guess a set would be better suited?

Comment (fulmicoton) [reply src/indexer/index_writer.rs]: ah I see. I guess that's ok.

**Comment by PSeitz** [comment]
fixes #1089

**Pull request merged**